packages/components/tests/dummy/app/templates/components/reveal.hbs

```diff
@@ -0,0 +1,167 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Reveal Component"}}
+
+<Shw::Text::H1>Reveal</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Content</Shw::Text::H2>
+
+  <Shw::Text::H3>On its own</Shw::Text::H3>
+
+  <Shw::Flex {{style gap="1.5rem"}} @direction="column" as |SF|>
+    <SF.Item @label="With plain text content">
+      <Hds::Reveal @text="Developer instructions">
+        Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to
+        corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the
+        holistic world view of disruptive innovation via workplace diversity and empowerment.
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With HTML formatted content (containing alert & list)">
+      <Hds::Reveal @text="Show me how">
+        <Hds::Alert @type="compact" as |A|>
+          <A.Description>Instructions to add the verification record to your domain host</A.Description>
+        </Hds::Alert>
+        <ol>
+          <li>
+            For the record type, select
+            <b>TXT</b>.
+          </li>
+          <li>
+            In the
+            <b>Name/Host/Alias</b>
+            field, enter
+            <b>@</b>
+            or leave blank. Your host might require you to enter your domain, which looks like
+            <b>example.com</b>
+          </li>
+          <li>
+            In the
+            <b>Time to Live (TTL)</b>
+            field, enter
+            <b>86400</b>
+            or leave the default.
+          </li>
+          <li>
+            In the
+            <b>Value/Answer/Destination</b>
+            field, paste the verification record you copied.
+          </li>
+          <li>
+            Save the record.
+          </li>
+        </ol>
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With interactive content">
+      <Hds::Reveal @text="More options">
+        See here for more info:
+        <a href="https://www.hashicorp.com/" target="_blank" rel="noopener noreferrer">HashiCorp</a>
+      </Hds::Reveal>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H3>Used next to other content</Shw::Text::H3>
```

Comment:
@cveigt Have you guys considered/discussed setting a default margin above/below the `Reveal` to ensure elements have some space between them? This would be based on the guidance you intend to include in the documentation, as was noted in Figma. I could see this being helpful so long as they can easily override the value for special use cases.

Reply:
I didn't consider having margin built into the component, if that's what you mean. Have we done that in other components? cc @KristinLBradley

Reply:
Adding default margins to components is an anti-pattern in design systems. Let's try to avoid it as much as possible, leads to a lot of problems (that's why it has become an anti-pattern, because people thought it was a good idea, tried it, and found out it was not).

Reply:
Going to go ahead and resolve this thread as I agree it adds more potential problems than it solves to add default margins to this type of component.